roles/maps/tasks/download_large_file.yml

```
@@ -56,26 +56,53 @@
           size_disp = str(round(size / MiB, 2)) + ' MiB'
         else:
           size_disp = str(round(size / GiB, 2)) + ' GiB'
-        print(' (' + size_disp + ') To check progress, run: "tail {{ working_dir }}/{{ log_file }}"')
+        print('(' + size_disp + ') To check download progress see above.')
       args:
         executable: /usr/bin/python3
       vars:
         # Where the file eventually goes
         dest_path: "{{ dest_base_path }}/{{ item }}"
 
-        log_file: "{{ item }}.log"
+        # Use md5sum to generate the log file name:
+        # * Name has short, consistent length. The "to check progress..." message
+        #   fits within 80 characters on terminal (at least in the current
+        #   ansible version) in case users have small screens.
+        # * If the user mis-copies, it's less likely to do anything destructive.
+        log_file: "{{ item | hash('md5') }}.log"
```

```suggestion
        log_file: "{{ item | hash('sha1') | truncate(8, True, '') }}.log"
```

sha1 has better mixing if you wanted to truncate

---

eh I guess it's fine either way

---

I was thinking about it but I don't know the properties of md5sum, at which point you start to get collisions. I suppose it's quite unlikely, plus somewhat unlikely that two such files would be present in this directory at once.